### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by westlakem** [swimlane/ngx-charts#609]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[X] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
The graph is tiny on mobile, and cuts off wording and the legend almost completely
![image](https://user-images.githubusercontent.com/4505074/31852135-76abf6a2-b640-11e7-802f-ad31b8d494bd.png)

**Expected behavior**
The graph should grow to match the "normal" text size of the page.  You may have to scroll to the right (off the screen like tables) but I feel that is better than having an unreadable graph.

Easier solution: 
Graph should either look like the desktop version:
![image](https://user-images.githubusercontent.com/4505074/31852161-e93e44f4-b640-11e7-9ee9-e1641f9bbbc2.png)

or 
Slightly more complicated (to save space on mobile) 
You can flip the names up on end and shrink the graph part slightly

![image](https://user-images.githubusercontent.com/4505074/31852193-812dd7d4-b641-11e7-8593-60e4c8597c21.png)

**Reproduction of the problem**
Create any graph, open it in google chrome, and emulate on a phone sized display.

**What is the motivation / use case for changing the behavior?**
Mobile First Design

**Please tell us about your environment:**
Angular 4 - Node 8

* **ngx-charts version:** x.x.x
    "@swimlane/ngx-charts": "^6.1.0",

* **Angular version:** 2.x.x
    "@angular/core": "^4.2.4",

* **Browser:** 
Any Mobile Devices

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
N/A

**Comment by Moulde** [comment]
Basically more responsive?

**Comment by westlakem** [comment]
@Moulde  yes, I also noticed that If I resize, or change values on the graph, it changes the size as well, and not in a good way.

**Comment by felix9607** [comment]
@westlakem the resize issue is sth discussed [here](https://github.com/swimlane/ngx-charts/issues/606) already. maybe it helps
